pycheribuild/projects/build_qemu.py

```diff
@@ -72,6 +72,8 @@ def setup_config_options(cls, **kwargs):
 
         cls.gui = cls.add_bool_option("gui", show_help=False, default=False,
                                       help="Build a the graphical UI bits for QEMU (SDL,VNC)")
+        cls.profile = cls.add_bool_option("profile", show_help=False, default=False,
```

Comment:
```suggestion
        cls.profile = cls.add_bool_option("build-profiler", show_help=False, default=False,
```
maybe buid-profiler makes more sense? `--qemu/profile` sounds to me like the build will be profiled.

Reply:
Good point